api/api.go

```diff
@@ -423,16 +441,16 @@ func HandlerWithOptions(si ServerInterface, options ChiServerOptions) http.Handl
 	}
 
 	r.Group(func(r chi.Router) {
-		r.Get(options.BaseURL+"/dev/ckmeans", wrapper.GetDevCkmeans)
+		r.Get(options.BaseURL+"/ckmeans/{year}", wrapper.GetCkmeansYear)
 	})
 	r.Group(func(r chi.Router) {
-		r.Get(options.BaseURL+"/dev/ckmeansratio", wrapper.GetDevCkmeansratio)
+		r.Get(options.BaseURL+"/ckmeansratio/{year}", wrapper.GetCkmeansratioYear)
 	})
 	r.Group(func(r chi.Router) {
-		r.Get(options.BaseURL+"/dev/hello/{dataset}", wrapper.GetDevHelloDataset)
+		r.Get(options.BaseURL+"/metadata", wrapper.GetMetadata)
```

Comment:
OOS for this PR, but @stmuk - reckon a `year` parameter for metadata will be needed eventually?

Reply:
yes